### PR TITLE
added template for cisco xr : "admin show environment fan" ,"admin show vm" and "show version"

### PR DESCRIPTION
**Pull request by rishabh5j**
##### ISSUE TYPE
 - New Template Pull Request

##### COMPONENT
cisco_xr_admin_show_environment_fan.template
cisco_xr_admin_show_vm.template
cisco_xr_show_version.template

##### SUMMARY
Added template cisco_xr_admin_show_environment_fan.template
Added template cisco_xr_admin_show_vm.template , 
Added template cisco_xr_show_version.template ,
Modified the index 
Added tests for all three

```

### CODE REVIEW

**Comment by rishabh5j** [comment]
@jedelman8  Hi Jason, can this be approved and merged?

File: templates/cisco_xr_admin_show_environment_fan.template [hunk 1]
```diff
@@ -0,0 +1,39 @@
+Value MODULE (0\/(FT|PM)\d)
```
Comment (itdependsnetworks) [inline templates/cisco_xr_admin_show_environment_fan.template]: what is wrong with `/S+` here?

File: templates/cisco_xr_admin_show_vm.template [hunk 2]
```diff
@@ -0,0 +1,21 @@
+Value Filldown LOCATION (\d\/(FC|RP|SC)?\d)
```
Comment (itdependsnetworks) [inline templates/cisco_xr_admin_show_vm.template]: Same here, what is wrong with `/S+`?

File: templates/cisco_xr_admin_show_environment_fan.template [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  ^\s+${FAN_SPEED} -> Continue
+  ^\s+\d+\s+${FAN_SPEED} -> Continue
+  ^\s+\d+\s+\d+\s+${FAN_SPEED} -> Continue
+  ^\s+\d+\s+\d+\s+\d+\s+${FAN_SPEED} -> Continue
+  ^\s+\d+\s+\d+\s+\d+\s+\d+\s+${FAN_SPEED} -> Continue
+  ^\s+\d+\s+\d+\s+\d+\s+\d+\s+\d+\s+${FAN_SPEED} -> Record
+  ^.*$$ -> PM
+  ^\s+$$
+  ^$$
+  ^.* -> Error "LINE NOT FOUND"
```
Comment (itdependsnetworks) [inline templates/cisco_xr_admin_show_environment_fan.template]: Thanks!

File: templates/cisco_xr_admin_show_environment_fan.template (hunk 1 above)
Comment (rishabh5j) [reply templates/cisco_xr_admin_show_environment_fan.template]: @itdependsnetworks  I though the regex should be specific if possible. Is that not correct?

Comment (itdependsnetworks) [reply templates/cisco_xr_admin_show_environment_fan.template]: To an extent yes, to an extent no :) 

Looking at the data, it doesn't seem like that would cover all future use cases. You are fairly well protected here in that you properly transition state to `FM`, after which I think it's a safe assumption that this is the only thing that would fit, and not be a false positive.

**Comment by itdependsnetworks** [comment]
If you make these few updates, can get this one in as well.

**Comment by jmcgill298** [comment]
https://github.com/rishabh5j/ntc-templates/pull/2 submitted to use "\S+"

**Pull request merged**